util/ranger/detacher.go

```diff
@@ -114,7 +116,7 @@ func getEqOrInColOffset(expr expression.Expression, cols []*expression.Column) i
 			offset = curOffset
 		}
 		return offset
-	case ast.EQ, ast.NullEQ:
+	case ast.EQ, ast.NullEQ, ast.LE, ast.GE:
```

Comment:
Can we add `ast.LT` and `ast.GT` here? It seems we can convert `x > 1 and x < 3` to `x = 2`?